entrypoint.sh

```diff
@@ -33,7 +33,7 @@ function main {
 
       file_dirname=$(dirname "${file}")
       file_basename=$(basename "${file}")
-      file_name="${file_basename%.*}"
+      #file_name="${file_basename%.*}"
```

Comment:
How does this differ from what you're doing in #22 to get a basename to create a brand new file?

Reply:
This is just cleanup so nobody wonders where `file_name` gets used.

Reply:
Let's just delete it then rather than uncomment. It exists in the git history.

Reply:
Done.